doc/API.md

```diff
@@ -476,17 +476,19 @@ This example first includes all `wcag2a` and `wcag2aa` rules. All rules that are
 
 6. Only process certain types of results
 
-The `resultTypes` option can be used to limit the result types that axe will process, aggregate, and send to the reporter. This can be useful for improving performance on very large or complicated pages when you are only interested in certain types of results.
+The `resultTypes` option can be used to limit the number of related nodes for a rule to a maximum of one. This can be useful for improving performance on very large or complicated pages when you are only interested in certain types of results.
```

Comment:
I think this should be "number of nodes for a rule to a maximum of one". It doesn't filter the related nodes, but the nodes that is returned. The same is true for all uses of "related nodes" below. If you just take out the word "related", this is correct. See:

https://github.com/dequelabs/axe-core/blob/develop/lib/core/reporters/helpers/process-aggregate.js#L63